manifests/a/AmN/IconExtractor/1.0.0/AmN.IconExtractor.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: AmN.IconExtractor
+PackageVersion: 1.0.0
+InstallerLocale: en-US
+InstallerType: msi
+ProductCode: '{671672BC-9824-4AA1-A25D-59DF6322EB90}'
+AppsAndFeaturesEntries:
+- DisplayName: Icon Extractor
+  Publisher: AmN
+  DisplayVersion: 1.0.0
+  ProductCode: '{671672BC-9824-4AA1-A25D-59DF6322EB90}'
+  UpgradeCode: '{D8B51DA0-15EF-4D07-BDF6-4B022570A9C2}'

```

```suggestion
- ProductCode: '{671672BC-9824-4AA1-A25D-59DF6322EB90}'
  UpgradeCode: '{D8B51DA0-15EF-4D07-BDF6-4B022570A9C2}'
```
These fields should be omitted since they contain the exact same information as other fields